exp/api/v1beta1/ocimanagedmachinepool_types.go

```diff
@@ -59,6 +59,10 @@ type OCIManagedMachinePoolSpec struct {
 	// +optional
 	InitialNodeLabels []KeyValue `json:"initialNodeLabels,omitempty"`
 
+	// NodePoolCyclingDetails defines the node pool recycling options.
+	// +optional
+	NodePoolCyclingDetails *NodePoolCyclingDetails `json:"nodePoolCyclingDetails,omitempty"`
```

Comment:
So CAPI will remove these fields since it knows about only v1beta1, so it was not working for me. I think for any fields which CAPI also mutates, we will need to add to both till CAPI moves to v1beta2.

Reply:
So scratch my first comment, I did not do conversion properly. Removed from v1beta1.